sites/all/themes/walkhub_theme/templates/page.tpl.php

```diff
@@ -359,7 +359,8 @@ class="fa fa-envelope-o"></i></a></li>
 
     <ul>
       <li>WalkHub &copy; 2014</li>
-      <li><a href="#">Terms &amp; conditions</a></li>
+      <li><a href="/node/3246">Terms &amp; conditions</a></li>
+      <li><a href="http://walkhub.net/content/community-champions-2nd-indiegogo-campaign">Community champions</a></li>
```

Comment:
This shouldn't be an absolute link. Use `l(t('Community champions'), 'node/NNN');` instead of a hardcoded link.